boards/entrypoint.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -e 
+
+if [ -z "$AZURE_BOARDS_ORGANIZATION" ]; then
+    echo "AZURE_BOARDS_ORGANIZATION is not set." >&2
+    exit 1
+fi
+
+if [ -z "$AZURE_BOARDS_PROJECT" ]; then
+    echo "AZURE_BOARDS_PROJECT is not set." >&2
+    exit 1
+fi
+
+if [ -z "$AZURE_BOARDS_TOKEN" ]; then
+    echo "AZURE_BOARDS_TOKEN is not set." >&2
+    exit 1
+fi
+
+if [ -z "$GITHUB_EVENT_PATH" ]; then
+    echo "GITHUB_EVENT_PATH is not set." >&2
+    exit 1
+fi
+
+function work_items_for_issue {
+    vsts work item query --wiql "SELECT ID FROM workitems WHERE [System.Tags] CONTAINS 'GitHub' AND [System.Tags] CONTAINS 'Issue ${GITHUB_ISSUE_NUMBER}'" | jq '.[].id' | xargs
+}
+
+AZURE_BOARDS_TYPE="${AZURE_BOARDS_TYPE:-Feature}"
+AZURE_BOARDS_CLOSED_STATE="${AZURE_BOARDS_CLOSED_STATE:-Closed}"
```

Comment:
The out of the box template for Features don't seem to have a state named "Closed":

```
ERROR: The field 'State' contains the value 'Closed' that is not in the list of supported values.
```

Reply:
> Overnight, I realized that I was probably using an old team project, and indeed, upon creating a new one I now have "Done" as a state. So ignore this comment. ☝️

Just to confirm, you meant that Closed should be the state, right?

Reply:
> Just to confirm, you meant that Closed should be the state, right?

Yes, excuse my sloppy comment above.  I agree that "Closed" makes sense for the default here.

Reply:
Fixed, thanks!